docs/misc/sprite-animator.mdx

```diff
@@ -43,7 +107,10 @@ type CommonMeshProps = {
   matrixAutoUpdate?: boolean
 }
 
-type SpriteAnimatorProps = {
+/**
+ * Props for the SpriteAnimator component
+ */
+export type SpriteAnimatorProps = {
   /** The start frame of the animation */
```

Comment:
please also add this type of "jsdoc comment" in the component, so Storybook will add it to the props table:
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/f0ce3e0b-7cca-4dda-8448-8df0624c5fe7">

Reply:
added props comments